samples/msal-node-samples/ElectronSystemBrowserTestApp/src/CustomLoopbackClient.ts

```diff
@@ -59,12 +58,12 @@ export class CustomLoopbackClient implements ILoopbackClient {
                     res.end(errorTemplate || "Error occurred loading redirectUrl");
                     reject(new Error('Loopback server callback was invoked without a url. This is unexpected.'));
                     return;
-                } else if (url === CommonConstants.FORWARD_SLASH) {
+                } else if (url === "/") {
```

Comment:
Are you removing the const as well?

Reply:
No, the const is used internally in msal-common/browser/node. I just don't want to demonstrate consuming things directly from common in our samples (and this isn't something we need to or should export from any of our libs)